src/main/java/code/practice/tasks/Room.java

```diff
@@ -0,0 +1,66 @@
+package code.practice.tasks;
+
+import java.util.Objects;
+
+public class Room {
```

Comment:
This class represents Room entity and doesn't contain any special logic so it should be moved to another package, like **model**.